ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -286,8 +287,30 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
     }
   }
 
-  @Explanation("Authentication errors.")
+  @Explanation("Authentication and authorization errors.")
   object AuthorizationChecks extends ErrorGroup() {
+
+    @Explanation("""The stream was aborted because the authenticated user's rights changed,
+        |and the user might thus no longer be authorized to this stream.
+        |""")
+    @Resolution(
+      "The application should automatically retry fetching the stream. It will either succeed, or fail with an explicit denial of authentication or permission."
+    )
+    object StaleUserManagementBasedStreamClaims
+        extends ErrorCode(
+          id = "STALE_STREAM_AUTHORIZATION",
+          ErrorCategory.ContentionOnSharedResources,
+        ) {
+      case class Reject()(implicit
+          loggingContext: ContextualizedErrorLogger
+      ) extends LoggingTransactionErrorImpl("Stale stream authorization. Retry quickly.") {
+        override def retryable: Option[ErrorCategoryRetry] = Some(
+          ErrorCategoryRetry(who = "", duration = 0.seconds)
```

Comment:
```suggestion
          ErrorCategoryRetry(who = "user", duration = 0.seconds)
```

Reply:
Afaict, value of `who` is not used anywhere